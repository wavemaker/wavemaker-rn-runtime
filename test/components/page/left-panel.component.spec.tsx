import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmLeftPanel from '@wavemaker/app-rn-runtime/components/page/left-panel/left-panel.component';

describe('Test LeftPanel component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmLeftPanel name="test_LeftPanel"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});