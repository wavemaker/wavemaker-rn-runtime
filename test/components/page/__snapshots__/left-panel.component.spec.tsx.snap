// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test LeftPanel component should render the left-panel component 1`] = `
[
  <View
    onLayout={[Function]}
    style={
      {
        "height": 0,
        "width": 0,
      }
    }
  />,
  <RCTScrollView
    collapsable={false}
    contentContainerStyle={
      [
        {
          "backgroundColor": "#ffffff",
          "borderBottomRightRadius": 16,
          "borderTopRightRadius": 16,
          "elevation": 1,
          "maxWidth": 360,
          "minHeight": "100%",
        },
        {
          "maxWidth": "100%",
          "width": "100%",
        },
      ]
    }
    handlerTag={1}
    handlerType="NativeViewGestureHandler"
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onScroll={[Function]}
    scrollEventThrottle={48}
    waitFor={
      [
        {
          "current": null,
        },
      ]
    }
  >
    <View />
  </RCTScrollView>,
]
`;
